src/it/JENKINS-45740-metadata/pom.xml

```
@@ -46,9 +48,9 @@
     </pluginRepositories>
 
     <scm>
-      <connection>scm:git:git://github.com/jenkinsci/${project.artifctId}-plugin.git</connection>
-      <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
-      <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-      <tag>HEAD</tag>
+        <connection>scm:git:git://github.com/jenkinsci/${project.artifctId}-plugin.git</connection>
```

Since GitHub is deprecating git:// protocol, we should use https:// for read access to repositories.
```suggestion
        <connection>scm:git:https://github.com/jenkinsci/${project.artifctId}-plugin.git</connection>
```

---

Thanks Mark. Good to update these one by one as we see them, but the whole ecosystem could use a broad sweep. How I wish I had a tool to do that. :smile: